### PR TITLE
Fix 2021.3.3

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
@shyuep Can you take a look at this pull request? And in case you need help with the maintenance, I opened a separate pull request to add myself as maintainer https://github.com/conda-forge/monty-feedstock/pull/28

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,15 +12,15 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
```
Comment (jan-janssen) [inline recipe/meta.yaml:15]:
```suggestion
  number: 0
```

**Comment by jan-janssen** [comment]
@shyuep Can you merge this pull request?

**Comment by wolfv** [comment]
@jan-janssen do you want to add yourself to the maintainers in this PR? I am going to merge it.

**Comment by jan-janssen** [comment]
@wolfv I noticed that I accidentally already removed my fork, so I opened a new pull request which includes all the changes. Can you merge https://github.com/conda-forge/monty-feedstock/pull/29 ?

**Pull request closed without merging**